magefile.go

```diff
@@ -10,10 +10,9 @@ import (
 	"os"
 	"path/filepath"
 
+	"github.com/elastic/package-registry/packages"
```

Comment:
nit: you might need sort imports here

Reply:
Oops, sorry, I merged the one for staging, I will open a PR to fix it there.